modelina-cli/test/integration/bundle/bundle.test.ts

```diff
@@ -0,0 +1,102 @@
+import { expect, test } from '@oclif/test';
```

Comment:
All of these non existing commands can just be removed completely 🙂 Only need the Modelina test and generic tests for context :v: